pkg/table/tables/tables.go

```diff
@@ -1130,7 +1130,6 @@ func genIndexKeyStr(colVals []types.Datum) (string, error) {
 func (t *TableCommon) addIndices(sctx sessionctx.Context, recordID kv.Handle, r []types.Datum, txn kv.Transaction, opts []table.CreateIdxOptFunc) (kv.Handle, error) {
 	writeBufs := sctx.GetSessionVars().GetWriteStmtBufs()
 	indexVals := writeBufs.IndexValsBuf
-	skipCheck := sctx.GetSessionVars().StmtCtx.BatchCheck
```

Comment:
Why remove this? Seems `batchCheck` maybe useful. /cc @tangenta

Reply:
This may affect the behavior of `INSERT IGNORE` statements. It is better to keep this code in line 1147.

Reply:
I think `skipCheck` means no need check. While the code return `kv.ErrKeyExists`, it did check, so think the code maybe have outdated logic. Maybe we can ask @tangenta for help to confirm this.

Reply:
> When [INSERT IGNORE](https://dev.mysql.com/doc/refman/8.0/en/insert.html) is used, the insert operation fails silently for rows containing the unmatched value, but inserts rows that are matched.

Reply:
The `StmtCtx.BatchCheck` is also used elsewhere, it's better to improve or refactor in another PR if it's not related to the message fixing to avoid introducing extra risks.

Reply:
`skipCheck ` is reserved as suggest.[77c05bd7a5ebc69fbdf65482459efdea01b93050](https://github.com/pingcap/tidb/pull/46608/commits/77c05bd7a5ebc69fbdf65482459efdea01b93050)